todo_app/Lib/site-packages/Django-3.1.6.dist-info/INSTALLER

```diff
@@ -0,0 +1 @@
+pip
```

Comment:
all the `/Lib/` files should be added to`.gitignore`

Reply:
added